### PR TITLE
[IMP] mail: introduce RtcController/headphoneButtonTitle

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/rtc_controller.js [hunk 1]
```diff
@@ -74,6 +74,17 @@ registerModel({
                 startWithVideo: true,
             });
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeHeadphoneTitle() {
+            if (this.messaging.rtc.currentRtcSession.isDeaf) {
```
Comment (oomsveta) [inline addons/mail/static/src/models/rtc_controller.js:82]: You should ensure that `currentRtcSession` is defined before trying to access `isDeaf`.

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js:82]: Template has no guard, so I think it never happens having `RtcController` and no `currentRtcSession`?

Comment (oomsveta) [reply addons/mail/static/src/models/rtc_controller.js:82]: I've tested it and it crashes

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js:82]: Oh ok, then I'll change it

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js:85]: Changes done

**Review by oomsveta — APPROVED** [review]

File: addons/mail/static/src/models/rtc_controller.js [hunk 2]
```diff
@@ -74,6 +74,20 @@ registerModel({
                 startWithVideo: true,
             });
         },
+        /**
+         * @private
+         * @returns {string}
```
Comment (oomsveta) [inline addons/mail/static/src/models/rtc_controller.js:79]: string or FieldCommand

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc_controller.js:79]: Changes done

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/90635

**Pull request closed without merging**